packages/client/scripts/populate-assetlinks.ts

```diff
@@ -25,12 +25,16 @@ Ethereal Engine. All Rights Reserved.
 
 import appRootPath from 'app-root-path'
 import cli from 'cli'
-import fs from 'fs'
 import fetch from 'node-fetch'
+import path from 'path'
 
-import { writeFileSyncRecursive } from '@etherealengine/common/src/utils/fsHelperFunctions'
+import { writeFileAsync } from '@etherealengine/common/src/utils/fsHelperFunctions'
 
-const envPath = appRootPath.path + `/packages/client/public/.wellknown/assetlinks.json`
+// Deprecated in favour of a safer approach, using path.join will resolve accounting the OS,
+// to concatente path strings is a well known bad practice in Node.js
+// --> const envPath = appRootPath.path + `/packages/client/public/.wellknown/assetlinks.json`
```

Comment:
these deprecation notices are unnecessary, we can just change it and not document the old code

Reply:
Documenting deprecated code is communication strategy that provides information such a the reason and the alternative, it's useful information that allow developers to better understand the changes and as a reference for future implementations, for now I'm simply addressing your change request but it's important to understand the importance of keeping deprecated code documented.

![image](https://github.com/EtherealEngine/etherealengine/assets/1533239/8df06e6f-f857-4cbb-9238-c0828cd86346)

Reply:
For our own code and endpoint APIs I agree, but this is a simple refactor. We should define what we should do in these situations in our standards / best practices documentation .